docs/README.md

```diff
@@ -6,40 +6,110 @@
 
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-The **Storefront Permissions UI** is the interface for the APP [`vtex.storefront-permissions`](https://github.com/vtex-apps/storefront-permissions)
+**Storefront Permissions UI** provides an interface for the [Storefront Permissions](https://developers.vtex.com/vtex-developer-docs/docs/vtex-storefront-permissions) app, which is part of VTEX’s [B2B Suite](https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-suite) solution, a collection of apps that allow stores to manage organizations, storefront roles and permissions, and checkout settings for B2B commerce relationships.
 
-![image](https://user-images.githubusercontent.com/24723/138930342-0b32937f-120b-49c9-9d3b-a20e350450a1.png)
+B2B customers often play distinct roles within their organization, such as professional buyer, manager, or supervisor. Each role is associated with a different set of permissions, depending on the actions the user needs to perform.
 
-## Functionalities
+This app communicates with the backstage **Storefront Permissions** app and provides the following features for this scenario:
 
-- Manage Roles
-- Manage Users
-- Manage theme blocks depending on user's role
+* [Roles management](#roles-management): allow VTEX Admin users to manage B2B roles and associated app permissions through an interface.
+* [Theme blocks configuration](#theme-blocks-configuration): enable conditional theme blocks so that only users with the required permissions can access determined parts of the content in your storefront.
 
-## Configuration
+We recommend that you use it alongside the other apps in the [B2B Suite](https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-suite) for all functionalities to work as expected.
 
-1. [Install](https://vtex.io/docs/recipes/development/installing-an-app/) the storefront-permissions app by running `vtex install vtex.storefront-permissions-ui` in your terminal.
-2. At the Admin, navigate to **ACCOUNT SETTINGS > Storefront Permissions**.
 
-3. Create Users
+## Before you start
 
-### Theme configuration
+Make sure you have the [VTEX IO CLI (Command Line Interface)](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-vtex-io-cli-install) installed in your machine.
 
-To make use of the conditional blocks, you need to add `vtex.storefront-permissions-ui@1.x` under `dependencies` on the `manifest.json`
+You must also have installed the [Storefront Permissions](https://developers.vtex.com/vtex-developer-docs/docs/vtex-storefront-permissions) app to use the **Storefront Permissions UI**.
```

Comment:
This isn't strictly true. Installing Storefront Permissions UI will have the effect of automatically installing Storefront Permissions as well (as a dependency).

Reply:
Fixed ✅ Added this to the **Installation** section:

> ℹ Installing **Storefront Permissions UI** has the effect of automatically installing the [Storefront Permissions](https://developers.vtex.com/vtex-developer-docs/docs/vtex-storefront-permissions) app as a dependency, in case it is not already installed.